assignments/objects.js

```diff
@@ -49,7 +103,30 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = [
```

Comment:
Good job attempting the stretch. Its a little bit off. Look at the solution code and see what changes you have to make.